### ISSUE TITLE
False error for numpy module

### ISSUE THREAD

**Issue by zyrikby** [PyCQA/pylint#1477]
### Steps to reproduce
```python
import numpy
angle = numpy.arctan(1/2)
```

### Current behavior
Module 'numpy' has no 'arctan' member; maybe 'arctan2'? (no-member)

### Expected behavior
arctan is a member of numpy module.

### pylint --version output
No config file found, using default configuration
pylint 1.7.1,
astroid 1.5.2
Python 3.6.1 (default, May  9 2017, 18:46:04)
[GCC 5.4.0 20160609]

**Comment by rogalski** [comment]
Thanks for submitting an issue!

**Comment by nschloe** [comment]
The same is true for a number of other numpy functions; `logical_not`, `minimum`, etc.

**Comment by rogalski** [comment]
It's likely a misimplemented [brain_numpy.py](https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_numpy.py)

**Comment by hippo91** [comment]
Should we transfer this bug report into astroid?

**Comment by hippo91** [comment]
Beginning of a fix proposed through PR #449 inside astroid repository.

**Comment by PCManticore** [comment]
Solved by @hippo91's patch.
